crud/helpers.go

```diff
@@ -191,9 +191,16 @@ func stateRefreshFunc(sync StatefulResource) resource.StateRefreshFunc {
 
 func waitForStateRefresh(sync StatefulResource, pending, target []string) (e error) {
 	timeoutStr := os.Getenv("TF_VAR_timeout_minutes")
```

Comment:
You're right, this change should have just pulled the value from the provider instead of continuing to grab it from the env var. good catch!